lib/CountrySelection/CountrySelection.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import countries from '../../util/countries';
+
+import Selection from '../Selection';
+
+const CountrySelection = props => {
+  const intl = useIntl();
+
+  const sortedCountries = countries.map(c => {
+    return {
+      label: intl.formatDisplayName(c.alpha2, { type: 'region' }),
+      value: c.alpha2,
+    };
+  }).sort((a, b) => a.label.localeCompare(b.label));
```

Comment:
[useMemo](https://reactjs.org/docs/hooks-reference.html#usememo)
```suggestion
  const sortedCountries = useMemo(() => (
    countries
      .map(c => ({
        label: intl.formatDisplayName(c.alpha2, { type: 'region' }),
        value: c.alpha2,
      })
      .sort((a, b) => a.label.localeCompare(b.label))
  ), countries)
```